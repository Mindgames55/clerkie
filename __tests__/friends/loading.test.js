import { render } from '@testing-library/react';
import Loading from '@/app/friends/loading';

it('renders Loading correctly', () => {
  const { container } = render(<Loading />);
  expect(container).toMatchSnapshot();
});