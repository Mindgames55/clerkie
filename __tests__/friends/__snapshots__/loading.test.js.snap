// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loading correctly 1`] = `
<div>
  <div
    class="container"
  >
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
    <span
      class="loader"
    />
  </div>
</div>
`;
